### ISSUE TITLE
v2raya应用设置导致的UDP转发问题/V2RAYA will auto disable "net.ipv4.ip_forward" config in sysctl.conf

### ISSUE THREAD

**Issue by ZhiShengYuan** [v2rayA/v2rayA#413]
My v2raya version is 1.6.5.2(as the link in GitHub is https://github.com/v2rayA/v2rayA/releases/download/v1.5.6.2/installer_debian_amd64_1.5.6.2.deb)
install method is direct install(use fhs install v2ray and deb package to install)
the operating system is Debian bullseye(a.k.a Debian 11),kernel version is 5.10.0-11-amd64
we found that the v2raya will auto-reset the "net.ipv4.ip_forward" after the setting is changed, this will cause the UDP forward to fail, the "IP Forward" and "Port Sharing" already disabled, and the Transparent Proxy/System Proxy Implementation is redirect(unsupport UDP)
So is there any way to disallow V2raya to change/rewrite this setting?

Possible steps to reproduce
1. install Debian 11
2. install UFW and pppoeconf
3. config PPPoE 
4. enable "net.ipv4.ip_forward"  in sysctl and 'sysctl -p',and use "ufw route" to set nat(like ufw route allow in on eno1 out on ppp0)
5. install v2ray-core(from v2fly) and v2raya(from deb)
6.  disable "IP Forward" and  "Port Sharing" 
7.  launch v2raya
8.  change any setting in v2raya
9.  apply, watch the UDP connectivity
10. if all goes well, the UDP forward will fail but the TCP is normal
11.  run sysctl -p as root,the UDP traffic should be normal

**Comment by ZhiShengYuan** [comment]
tl;dr 
使用v2raya 1.6.5.2(系统为debian 11,kernel version为 5.10.0-11-amd64)，当v2raya中不开启端口分享与IP转发时，v2raya疑似会覆写sysctl中ipv4转发设置以致于udp包无法转发(tcp正常),手动执行sysctl -p后恢复

**Comment by cubercsl** [comment]
@ZhiShengYuan v2raya 的 ip 转发就是这样实现的，你要使用 ip 转发在 v2raya里打开就行了。

**Comment by ZhiShengYuan** [comment]
> @ZhiShengYuan v2raya 的 ip 转发就是这样实现的，你要使用 ip 转发在 v2raya里打开就行了。

但是v2raya的ip转发有限制么(即允许且仅允许从内网(eno1)转发至外网(ppp0))

**Comment by mzz2017** [comment]
> > @ZhiShengYuan v2raya 的 ip 转发就是这样实现的，你要使用 ip 转发在 v2raya里打开就行了。
> 
> 
> 
> 但是v2raya的ip转发有限制么(即允许且仅允许从内网(eno1)转发至外网(ppp0))

v2raya 的 IP 转发就是 net.ipv4.ip_forward
